pkg/gofr/request.go

```diff
@@ -13,4 +15,5 @@ type Request interface {
 	PathParam(string) string
 	Bind(interface{}) error
 	HostName() string
+	QueryParams() queryparam.QueryParams
```

Comment:
since, we have added new methods to fetch queryParams, shouldn't we deprecate the previous method `Param`?

Reply:
sure will add the comment for deprecating `Params` method.